testdata/110_basic/.terraform.lock.hcl

```diff
@@ -0,0 +1,38 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/hashicorp/aws" {
+  version = "3.58.0"
+  hashes = [
+    "h1:kx8p0xV/mGa+7vnGOk02AenU/ZKxF08yT/0jynZgSQI=",
```

Comment:
I can't actually tell from these hashes (by eye) what exact platforms do they cover, but I suppose they should cover all 3 major platforms we support and test on:
 - darwin
 - windows
 - linux

`amd64` architecture should suffice.

I understand that this is not essential for the tests to pass on these platforms, but it would be great for reproducibility of the test data.

Reply:
Good call. Done using:

```
terraform providers lock \
  -platform=linux_amd64 -platform=darwin_amd64 -platform=windows_amd64 \
  registry.terraform.io/hashicorp/null registry.terraform.io/hashicorp/aws
```